src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* 'az containerapp add-on' : support for add-on weaviate create and delete commands
```

Comment:
Please note the breaking change on the history.rst  about if the customer is using the Azure CLI version greater than 2.56.1 , these commands will not work and cause breaking change.
For example:

* [Breaking Change]  'az containerapp service': deprecate from Azure CLI version 2.56.1
* 'az containerapp add-on' : support for add-on weaviate create and delete commands